extensions/ql-vscode/src/packaging.ts

```diff
@@ -9,18 +9,23 @@ import { ProgressCallback, UserCancellationException } from "./commandRunner";
 import { extLogger } from "./common";
 import { asError, getErrorStack } from "./pure/helpers-pure";
 import { redactableError } from "./pure/errors";
+import { QueryLanguage } from "./types/query-language";
 
-const QUERY_PACKS = [
-  "codeql/cpp-queries",
-  "codeql/csharp-queries",
-  "codeql/go-queries",
-  "codeql/java-queries",
-  "codeql/javascript-queries",
-  "codeql/python-queries",
-  "codeql/ruby-queries",
-  "codeql/csharp-solorigate-queries",
-  "codeql/javascript-experimental-atm-queries",
-];
+const PACKS_BY_QUERY_LANGUAGE = {
```

Comment:
This is definitely better to be using constants that define the different languages, but their use is still distributed throughout the code. Could you consolidate the different objects that contain the the languages in one area? This way, when we add new languages it will be easier to ensure that we've updated all the right blocks.

Reply:
I've opted to move this constant and the dbSchemeToLanguage constant into the same file as the type.